config/locales/en.yml

```diff
@@ -78,6 +78,7 @@ en:
   sign_in_github: Sign in with Github
   github_sign_in_failed: Github sign in failed
   api_exception: We apologize but we encountered an internal error. Please try again later or contact info@openhub.net if the problem persists.
+  kb_message: "Published message for project %{kb.project_id}"
```

Comment:
This won't work since you haven't passed `kb` object when calling `t(:kb_message)`.